tests/test_url_encode_keys_decorator.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# coding=utf8
+
+from simplekv.decorator import URLEncodeKeys
+from simplekv.memory import DictStore
+import pytest
+
+from basic_store import BasicStore
+
+
+class TestURLEncodeKeysDecorator(BasicStore):
+    @pytest.fixture()
+    def store(self):
+        base_store = DictStore()
+        return URLEncodeKeys(base_store)
+
+    def test_urlencode(self, store):
+        store.put(u'key special:-🍺', b'val1')
+        assert store.get(u'key special:-🍺') == b'val1'
```

Comment:
I think this test would be better if it would actually check the conversion. As it is now, it does not test anything that any `KeyValueStore` could not do, which is necessary (to prove that URLEncodeKeys works like any store), but not sufficient to test the actual functionality. My suggestion would be to have a separate test, not derived from `BasicStore`, which `.put()`s a key and checks the undecorated store for the properly encoded key and another one to test `_unmap_key()`.